docs/content/docs/dev/table/functions/systemFunctions.md

```diff
@@ -146,6 +146,7 @@ For Table API, please use `_` for spaces (e.g., `DAY_TO_HOUR`).
 |                          | `MICROSECOND`                  |
 | `DOY` _(SQL-only)_       |                                |
 | `DOW` _(SQL-only)_       |                                |
+| `EPOCH` _(SQL-only)_     |                                |
```

Comment:
You updated `TimeIntervalUnit.java`. So many symbols in this table are not `SQL-only` anymore?